client/components/header/index.js

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+import { NavLink } from "react-router-dom";
+import { SubNav } from "../subnav";
+import { BearLogo } from "./bearlogo";
+import BearsDoingStuff from "./bears-doing-stuff.png";
+import "./styles.css";
```

Comment:
You can just import this once in the App component then you don't need to import it into every component, one import will apply to the whole project.